### PR TITLE
Fixes #284 Building sync gateway fails due to repo restructure

### LINKED ISSUE #284: Building sync gateway fails due to repo restructure
https://gist.github.com/tleyden/f5e1119211602f6fd067#file-gistfile1-txt-L442-L444

I'm pretty sure that's due to changes in the way we are structuring the repo

### PR DESCRIPTION
**Pull request by tleyden**
https://github.com/couchbaselabs/sync-gateway-testcluster/issues/284

### CODE REVIEW

File: provision/ansible/playbooks/install-sync-gateway-source.yml [hunk 1]
```diff
@@ -42,38 +43,37 @@
...
-    shell: git checkout {{ branch }} chdir=/home/centos/sync_gateway
-  - name: update submodules
-    shell: git submodule init && git submodule update chdir=/home/centos/sync_gateway
-
-  # Build with flags if flags are defined, by default build_flags=""
-  - debug: msg="Building with flags {{ build_flags }}"
-  - name: build sync_gateway
-    shell: ./build.sh {{ build_flags }} chdir=/home/centos/sync_gateway
-
+  - name: install repo tool
```
Comment (sethrosetter) [inline provision/ansible/playbooks/install-sync-gateway-source.yml]: @tleyden Is this the best place for this or should it be install as part of the common tools?

Comment (tleyden) [inline provision/ansible/playbooks/install-sync-gateway-source.yml]: agreed, common tools is a better place

**Comment by tleyden** [comment]
Ok repushed with `repo` install in common tools

File: provision/ansible/playbooks/install-sync-gateway-source.yml [hunk 2]
```diff
@@ -42,38 +43,35 @@
...
-    git: repo=https://github.com/couchbase/sync_gateway.git
-         dest=/home/centos/sync_gateway
-         recursive=no
-  - name: checkout feature branch
-    shell: git checkout {{ branch }} chdir=/home/centos/sync_gateway
-  - name: update submodules
-    shell: git submodule init && git submodule update chdir=/home/centos/sync_gateway
-
-  # Build with flags if flags are defined, by default build_flags=""
-  - debug: msg="Building with flags {{ build_flags }}"
```
Comment (sethrosetter) [inline provision/ansible/playbooks/install-sync-gateway-source.yml]: @tleyden Does this mean passing flags to the build is no long supported?

Comment (tleyden) [inline provision/ansible/playbooks/install-sync-gateway-source.yml]: good catch!  I will fix that

**Comment by tleyden** [comment]
Fixed build flags issue

**Comment by sethrosetter** [comment]
@tleyden Thanks!

**Pull request merged**